app/authentication/controllers/authentication_controller.ts

```diff
@@ -32,4 +36,60 @@ export default class AuthenticationController {
 
     return response.redirect().toRoute('home')
   }
+
+  async forgotPassword({ inertia }: HttpContext) {
+    return inertia.render('manager/authentication/forgot_password_page')
+  }
+
+  async forgotPasswordAction({ request, response }: HttpContext) {
+    // const resend = new Resend('resend')
+    const email = request.only(['email']).email
+
+    const userByEmail = await User.query().where('email', email).first()
+
+    if (userByEmail) {
+      const resetEmailUrl = router
+        .builder()
+        .prefixUrl('http://localhost:3333')
```

Comment:
This cannot work in production because localhost doesn't exists from external client